src/patternfly/layouts/Gallery/styles.scss

```diff
@@ -1,7 +1,42 @@
+@import "../../patternfly-utilities";
+
+$pf-l-gallery__item--Width: 250px !default;
+
+$pf-l-gallery--GridGap: 1rem !default;
```

Comment:
this should be a the gutter global variable

Reply:
Done.